docs/src/references/c8y-configuration-management.md

```diff
@@ -272,7 +272,7 @@ the `c8y_configuration_plugin` service notifies this update over MQTT.
 Note that:
 * If no specific type has been assigned to a configuration file, then the path to this file is used as its type.
   Update notifications for that file are then published on the topic `tedge/configuration_change/{path}`,
-  for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
+  for instance `tedge/configuration_change/etc/tedge/mosquitto-conf/c8y-bridge.conf`.
```

Comment:
It looks weird, but it is not a mistake. The contract is `tedge/configuration_chage/{path}`. Since the file path is `/etc/...`, it publishes onto the topic `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`

Reply:
@rina23q Though I think this is a mistake that we are not stripping the leading `/` from the path when we are converting it to a topic. We don't support any relative paths in the configuration manager, so it seems a bit redundant, and looks very strange with the double slashes....though I guess it another topic so I will revert. Thanks for the info!